website/integrations/services/proxmox-ve/index.md

```diff
@@ -24,36 +24,63 @@ The following placeholders will be used:
 -   `proxmox.company` is the FQDN of the Proxmox VE server.
 -   `authentik.company` is the FQDN of the authentik install.
 
-### Step 1
+## authentik setup
 
-Under _Providers_, create an OAuth2/OpenID provider with these settings:
+1. **Navigate to the Admin interface, then click Providers to create an OAuth2/OpenID provider with these settings:**
 
--   Name: proxmox
--   Redirect URI: `https://proxmox.company:8006` (Note the absence of the trailing slash, and the inclusion of the webinterface port)
--   Signing Key: Select any available key
+    - **Name:** proxmox
+    - **Redirect URI:** `https://proxmox.company:8006` (No trailing slash, include the web interface port)
+    - **Signing Key:** Select any available key
 
-### Step 2
+2. **Create an application using this provider**
+    - Under **Applications** > **Applications** in the Admin interface, create a new application and configure it to use the provider created in the previous step.
+    - Optionally, apply access restrictions to the application.
+    - Set the **Launch URL** to `https://proxmox.company:8006`.
 
-Create an application which uses this provider. Optionally apply access restrictions to the application.
+## Proxmox VE setup (web interface)
 
-Set the Launch URL to `https://proxmox.company:8006`.
+1. **Log in to the Proxmox web interface**
 
-## Proxmox VE Setup
+    - Access the Proxmox VE web interface and log in with an administrative account.
 
-Proxmox VE allows configuration of authentication sources using the web interface (under Datacenter -> Permissions -> Realms).
+2. **Navigate to authentication source settings**
 
-![](proxmox-source.png)
+    - Go to **Datacenter** > **Permissions** > **Realms**.
+    - Click **Add** and select **Realm** to open the Add Realm dialog.
 
-Another way is to use the CLI. SSH into any Proxmox cluster node, and issue the following command:
+3. **Fill out the OpenID Connect settings**
 
-`pveum realm add authentik --type openid --issuer-url https://authentik.company/application/o/proxmox/ --client-id xxx --client-key xxx --username-claim username --autocreate 1`
+    - In the dialog that appears, fill in the following details:
+        - **Issuer URL**: Enter the Issuer URL from authentik (found under _Provider Metadata_), e.g., `https://authentik.company/application/o/proxmox/`.
```

Comment:
```suggestion
        - **Issuer URL**: Enter the Issuer URL from authentik (found under **Provider Metadata**), e.g., `https://authentik.company/application/o/proxmox/`.
```

Reply:
Where exactly is Issuer Metadata? Is that a field name somewhere on authentik's UI? Let's describe where to find this value a bit better.